From 6f0e0fa0de147cbc358968d27353d04b0fbc6daa Mon Sep 17 00:00:00 2001
From: Roman Stratiienko <r.stratiienko@gmail.com>
Date: Mon, 23 Oct 2023 01:27:32 +0300
Subject: [PATCH 01/18] cros_gralloc: Remove Android.bp files

Android.bp files are maintained in the AOSP fork repository.

Change-Id: I492e0c2154b22cd4cda85e379df2578b352c23df
Signed-off-by: Roman Stratiienko <r.stratiienko@gmail.com>
---
 cros_gralloc/aidl/Android.bp           | 51 --------------------------
 cros_gralloc/mapper_stablec/Android.bp | 44 ----------------------
 2 files changed, 95 deletions(-)
 delete mode 100644 cros_gralloc/aidl/Android.bp
 delete mode 100644 cros_gralloc/mapper_stablec/Android.bp

diff --git a/cros_gralloc/aidl/Android.bp b/cros_gralloc/aidl/Android.bp
deleted file mode 100644
index da8f670..0000000
--- a/cros_gralloc/aidl/Android.bp
+++ /dev/null
@@ -1,51 +0,0 @@
-//
-// Copyright (C) 2022 The Android Open Source Project
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package {
-    // See: http://go/android-license-faq
-    // A large-scale-change added 'default_applicable_licenses' to import
-    // all of the 'license_kinds' from "external_minigbm_license"
-    // to get the below license kinds:
-    //   SPDX-license-identifier-Apache-2.0
-    //   SPDX-license-identifier-BSD
-    default_applicable_licenses: ["external_minigbm_license"],
-}
-
-cc_binary {
-    name: "android.hardware.graphics.allocator-service.minigbm",
-    defaults: ["minigbm_cros_gralloc_defaults"],
-    relative_install_path: "hw",
-    init_rc: ["allocator.rc"],
-    vintf_fragments: ["allocator.xml"],
-    vendor: true,
-    shared_libs: [
-        "android.hardware.graphics.allocator-V2-ndk",
-        "android.hardware.graphics.mapper@4.0",
-        "libbase",
-        "libbinder_ndk",
-        "libgralloctypes",
-        "libhidlbase",
-        "liblog",
-        "libminigbm_gralloc",
-        "libminigbm_gralloc4_utils",
-    ],
-    static_libs: [
-        "libaidlcommonsupport",
-    ],
-    srcs: [
-        "Allocator.cpp",
-        "Main.cpp",
-    ],
-}
diff --git a/cros_gralloc/mapper_stablec/Android.bp b/cros_gralloc/mapper_stablec/Android.bp
deleted file mode 100644
index b25be17..0000000
--- a/cros_gralloc/mapper_stablec/Android.bp
+++ /dev/null
@@ -1,44 +0,0 @@
-//
-// Copyright (C) 2022 The Android Open Source Project
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package {
-    // See: http://go/android-license-faq
-    // A large-scale-change added 'default_applicable_licenses' to import
-    // all of the 'license_kinds' from "external_minigbm_license"
-    // to get the below license kinds:
-    //   SPDX-license-identifier-Apache-2.0
-    //   SPDX-license-identifier-BSD
-    default_applicable_licenses: ["external_minigbm_license"],
-}
-
-cc_library_shared {
-    name: "mapper.minigbm",
-    defaults: ["minigbm_gralloc4_common_defaults"],
-    vintf_fragments: ["mapper.minigbm.xml"],
-    shared_libs: [
-        "android.hardware.graphics.allocator-V2-ndk",
-        "libminigbm_gralloc",
-    ],
-    header_libs: [
-        "libbase_headers",
-        "libimapper_stablec",
-        "libimapper_providerutils",
-    ],
-    srcs: [
-        ":minigbm_gralloc4_mapper_files",
-        "Mapper.cpp",
-    ],
-    cpp_std: "c++20",
-}
-- 
2.39.2

